eng/Versions.props

```diff
@@ -10,22 +10,20 @@
     <!-- Latest symstore version updated by darc -->
     <MicrosoftSymbolStoreVersion>1.0.156301</MicrosoftSymbolStoreVersion>
     <!-- Runtime versions to test -->
-    <MicrosoftNETCoreApp21Version>2.1.22</MicrosoftNETCoreApp21Version>
+    <MicrosoftNETCoreApp21Version>2.1.23</MicrosoftNETCoreApp21Version>
     <MicrosoftAspNetCoreApp21Version>$(MicrosoftNETCoreApp21Version)</MicrosoftAspNetCoreApp21Version>
-    <MicrosoftNETCoreApp31Version>3.1.8</MicrosoftNETCoreApp31Version>
+    <MicrosoftNETCoreApp31Version>3.1.10</MicrosoftNETCoreApp31Version>
     <MicrosoftAspNetCoreApp31Version>$(MicrosoftNETCoreApp31Version)</MicrosoftAspNetCoreApp31Version>
-    <MicrosoftNETCoreApp50Version>
-    </MicrosoftNETCoreApp50Version>
-    <MicrosoftAspNetCoreApp50Version>
-    </MicrosoftAspNetCoreApp50Version>
+    <MicrosoftNETCoreApp50Version>5.0.0</MicrosoftNETCoreApp50Version>
+    <MicrosoftAspNetCoreApp50Version>$(MicrosoftNETCoreApp50Version)</MicrosoftAspNetCoreApp50Version>
     <!-- Latest shared runtime version updated by darc -->
-    <MicrosoftNETCoreAppVersion>5.0.0-rtm.20563.12</MicrosoftNETCoreAppVersion>
-    <MicrosoftNETCoreAppRuntimewinx64Version>5.0.0</MicrosoftNETCoreAppRuntimewinx64Version>
+    <MicrosoftNETCoreAppVersion></MicrosoftNETCoreAppVersion>
+    <MicrosoftNETCoreAppRuntimewinx64Version></MicrosoftNETCoreAppRuntimewinx64Version>
     <!-- Latest shared aspnetcore version updated by darc -->
-    <MicrosoftAspNetCoreAppRefInternalVersion>5.0.0-rc.2.20475.17</MicrosoftAspNetCoreAppRefInternalVersion>
-    <MicrosoftAspNetCoreAppRefVersion>5.0.0-rc.2.20475.17</MicrosoftAspNetCoreAppRefVersion>
```

Comment:
Are we ready to move to 6.0?

Reply:
I'm doing all of that in a separate PR. I didn't rebase and update the comment. I just pushed the update now.